### ISSUE TITLE
[Transforms] Incorrect emit of unicode in identifier

### ISSUE THREAD

**Issue by yuit** [Microsoft/TypeScript#8235]
**Original**
```ts
var constructorTestObject = new constructorTestClass(1, 'string', true, 2);
constructorTestObject.arg\u0031 = 1;
constructorTestObject.arg2 = 'string';
constructorTestObject.arg\u0033 = true;
constructorTestObject.arg4 = 2;
```
**Expect**
```js
var constructorTestClass = (function () {
    function constructorTestClass(arg1, arg\u0032, arg\u0033, arg4) {
        this.arg1 = arg1;
        this.arg\u0032 = arg\u0032;
        this.arg\u0033 = arg\u0033;
        this.arg4 = arg4;
    }
    return constructorTestClass;
}());
```
**Actual**
```js
var constructorTestClass = (function () {
    function constructorTestClass(arg1, arg\u0032, arg\u0033, arg4) {
        this.arg1 = arg1;
        this.arg2 = arg\u0032;
        this.arg3 = arg\u0033;
        this.arg4 = arg4;
    }
    return constructorTestClass;
}());
```
- escapedIdentifiers.js

**Comment by mhegazy** [comment]
duplicate of https://github.com/Microsoft/TypeScript/issues/8157?

**Comment by yuit** [comment]
@mhegazy it is the same issue. Though @rbuckton already fixes the example in #8157. I will just add this in the comment of #8157
